tests/foreman/ui/test_contentview.py

```diff
@@ -2708,6 +2708,7 @@ def test_positive_delete_with_kickstart_repo_and_host_group(session, default_sat
     os = os.update(['architecture', 'ptable'])
     with session:
         session.organization.select(org.name)
+        session.location.select('Any Location')
```

Comment:
I think by default its `Any location` why you need to explicitly set this ?

Reply:
@jyejare https://github.com/SatelliteQE/robottelo/issues/9040
@latran Can you please test with `smart_proxy_location.name`  as I did here https://github.com/SatelliteQE/robottelo/pull/9181

Reply:
@latran sorry, ignore my comment above, you do not use `smart_proxy_location` in that file, but what about this `session.location.select(constants.DEFAULT_LOC)` which I see used elsewhere in your module file.

Reply:
@swadeley I tested with `smart_proxy_location.name` and that worked

Reply:
oh, ok, good